lib/graphql/schema/field.rb

```diff
@@ -5,6 +5,8 @@
 module GraphQL
   class Schema
     class Field
+      NOT_CONFIGURED = ::Object.new
```

Comment:
Assuming you'd use this pattern in multiple classes, it might make sense to have a single `GraphQL::NOT_CONFIGURED`. You can also make it a `private_constant` if you don't want users to be able to pass it as argument.

Reply:
:+1: I added it here and I've opened https://github.com/rmosolgo/graphql-ruby/issues/4363 to track rolling it out across the codebase.